<?xml version="1.0"?>
<launch>

    <group ns="/ally1">

        <!-- Set ROS Params -->
        <param name="robot_estimator_on" type="bool" value="false" />
        <param name="go_rogue" type="bool" value="false" />

        <!-- Start the robot up -->
        <include file="$(find playground)/launch/robot.launch">
            <arg name="role" type="string" value="BIG_MAC" />
        </include>

    </group>
</launch>
